src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -169,9 +173,11 @@ private void experimentalInterpretation(final JavaSparkContext ctx,
                                             final Broadcast<SAMSequenceDictionary> broadcastSequenceDictionary,
                                             final String sampleId,
                                             final FindBreakpointEvidenceSpark.AssembledEvidenceResults assembledEvidenceResults) {
-        final String expOutputDir =
-                IOUtils.getPath(vcfOutputFileName).getParent().toAbsolutePath().toString() +
-                        "/experimentalVariantInterpretations";
+        String expOutputDir =
```

Comment:
It seems like it might be a lot cleaner to just replace the output file parameter with an output directory, and then write the requested files into that directory. Is that too much of a hassle? Alternatively you could just make the method of requesting the new variants be to provide a new file parameter, instead of a flag (for now, we can change this when we fully switch over).

Reply:
I actually thought about it, but eventually went the way it is currently implemented basically based on preference of "adding" over "replacing":
since it is still experimental, it might be better to clearly indicate that "in addition to what we have clearly evaluated/tested, we are intending to output more in the future, so here are both if you specifically know what you are requesting, with them separated into a single VCF and a directory, and please compare them if necessary"

For the alternative solution you are suggesting, is my understanding correct:
instead of a flag in the CLI, have a (possibly null) string, and the method `experimentalInterpretation(...)` could test if the string is null and call into the experimental code, so that user has the option of specifying the experimental output dir?

Reply:
Yes, that's what I was suggesting as an alternative. It seemed to match with how we handle some of our optional outputs, eg. you request writing of the assembly sam file or the target links file by passing in an output path for it. Does that make sense to you or does it seem weird?  This isn't really a big deal, feel free to choose whatever feels right to you. Hopefully we can get the old version of the tool replaced with the new version soon anyway.

Reply:
Yes, it totally makes sense. I'll make that change and merge afterwards.